snap/hooks/install

```diff
@@ -198,14 +230,3 @@ for svc in support-notifications support-scheduler support-logging app-service-c
     # stop them afterwards
     snapctl stop --disable "$SNAP_NAME.$svc"
 done
-
-# if we are on arm64, disable the security-proxy because kong isn't 
-# properly supported on arm64 due to incorrect memory pointers used by lua and
-# openresty
-# see https://github.com/edgexfoundry/blackbox-testing/issues/185 for more 
-# details
-if [ "$SNAP_ARCH" == "arm64" ]; then
-    snapctl set security-proxy=off
-    snapctl stop --disable "$SNAP_NAME.kong-daemon"
-    snapctl stop --disable "$SNAP_NAME.security-proxy-setup"
-fi
```

Comment:
hmm I just tried my kong snap on my rpi on arm64 and it didn't seem to work, so I think we need to leave this in for users to be able to use the snap on arm64 at all